### PR TITLE
add local ruby. prep for build automations.

### PR DESCRIPTION
**Pull request by shamilovtim**
prepping for automations. still blocked by block teams on build signing so i can move this forward.

- add local ruby to iOS with locked pods version
- bump cocoapods to latest
- add local ruby to Android 
- properly named .env example file

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
   <application android:name=".MainApplication" android:label="@string/app_name" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:allowBackup="false" android:theme="@style/AppTheme" android:usesCleartextTraffic="true">
     <meta-data android:name="expo.modules.updates.EXPO_UPDATE_URL" android:value="YOUR-APP-URL-HERE"/>
-    <meta-data android:name="expo.modules.updates.EXPO_SDK_VERSION" android:value="YOUR-APP-SDK-VERSION-HERE"/>
+    <meta-data android:name="expo.modules.updates.EXPO_SDK_VERSION" android:value="49.0.0"/>
```
Comment (amika-sq) [inline android/app/src/main/AndroidManifest.xml:24]: hehe, nice!

File: ios/Gemfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+ruby '~> 2.7.0'
+gem "fastlane"
+gem 'cocoapods', '1.14.2'
```
Comment (amika-sq) [inline ios/Gemfile:4]: Awesome! Just ran into this, where my installed cocoapods version was different.

File: ios/Gemfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+ruby '~> 2.7.0'
```
Comment (amika-sq) [inline ios/Gemfile:2]: In the project README, we callout 2.7.4 as the version for ruby. Should we update that?

Comment (shamilovtim) [reply ios/Gemfile]: fixed!

**Pull request merged**